mapcache.xml.sample

```diff
@@ -357,6 +357,11 @@
            preserve disk space.
       -->
       <symlink_blank/>
+      <!-- detect_blank
+          Similar to symlink_blank, except completely skip writing a file
+          if a blank, fully transparent tile is detected.
+      -->
```

Comment:
please add a note that this setting should only be used when using the seeder. if not (i.e. tiles generated on demand), this setting will cause mapcache to query the wms source each time this tile is requested